### ISSUE TITLE
Remove references to outdated media formats

### ISSUE THREAD

**Issue by Ryuno-Ki** [mdn/content#85]
While researching ARIA practices for an animation, I discovered, that 

https://github.com/mdn/content/blob/254b1adecd2e58173e1b0b8dd7b85b6a67d7d593/files/en-us/learn/accessibility/multimedia/index.html#L331-L339

is referencing Flash and Silverlight. As far as I know, both are dead (as in: no longer supported by their creators) and should not be mentioned in the docs.

Are there more places?

**Comment by Ryuno-Ki** [comment]
The respective MDN link is https://developer.mozilla.org/en-US/docs/Learn/Accessibility/Multimedia

**Comment by chrisdavidmills** [comment]
Agreed; we ought to fix this.

**Comment by Ryuno-Ki** [comment]
I looked through the files and found references to Silverlight here:
* [Polyfill on MDN](https://developer.mozilla.org/en-US/docs/Glossary/Polyfill) (to be deleted entirely?)

https://github.com/mdn/content/blob/f4c85bc27b29300bfbb7709e00e680d3f294b9eb/files/en-us/glossary/polyfill/index.html#L13

* Developer Guide (not on MDN?)

https://github.com/mdn/content/blob/f4c85bc27b29300bfbb7709e00e680d3f294b9eb/files/en-us/mozilla/developer_guide/introduction/obsolete_build_caveats_and_tips/index.html#L193

* Firefox 47 Release Notes (to be kept for historical reasons?)

https://github.com/mdn/content/blob/f4c85bc27b29300bfbb7709e00e680d3f294b9eb/files/en-us/mozilla/firefox/releases/47/index.html#L124

* [Cross-browser audio basics on MDN](https://developer.mozilla.org/en-US/docs/Web/Guide/Audio_and_video_delivery/Cross-browser_audio_basics)

https://github.com/mdn/content/blob/f4c85bc27b29300bfbb7709e00e680d3f294b9eb/files/en-us/web/guide/audio_and_video_delivery/cross-browser_audio_basics/index.html#L135

* [Audio and Video Delivery on MDN](https://developer.mozilla.org/en-US/docs/Web/Guide/Audio_and_video_delivery)

https://github.com/mdn/content/blob/main/files/en-us/web/guide/audio_and_video_delivery/index.html#L457

* [HTTP headers on MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers)

https://github.com/mdn/content/blob/f4c85bc27b29300bfbb7709e00e680d3f294b9eb/files/en-us/web/http/headers/index.html#L339

* [Accessible multimedia on MDN](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/Multimedia)

https://github.com/mdn/content/blob/f4c85bc27b29300bfbb7709e00e680d3f294b9eb/files/en-us/learn/accessibility/multimedia/index.html#L335-L346

* [Video and audio content on MDN](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Video_and_audio_content)

https://github.com/mdn/content/blob/f4c85bc27b29300bfbb7709e00e680d3f294b9eb/files/en-us/learn/html/multimedia_and_embedding/video_and_audio_content/index.html#L36

* [Apps CSP on MDN](https://developer.mozilla.org/en-US/docs/Archive/B2G_OS/Firefox_OS_apps/Building_apps_for_Firefox_OS/CSP)

(not in this repo?)

What to do with each of them?

**Comment by chrisdavidmills** [comment]
@Ryuno-Ki I've gone to work and updated all the instances you've reported here; see below for a description of what I've done. I decided to jump on top of this quickly, as it'll save me a lot of time later on (Flash is being removed in Fx 85...)

Thanks for your help here!

>     * [Polyfill on MDN](https://developer.mozilla.org/en-US/docs/Glossary/Polyfill) (to be deleted entirely?)

OK, fixed this by just providing a different example. It is still worth talking about what Polyfills are.

> The respective MDN link is https://developer.mozilla.org/en-US/docs/Learn/Accessibility/Multimedia

Removed the section, and just included the useful bit at the end of the article summary.

>     * Developer Guide (not on MDN?)

Hrm, yeah, I'm not sure where this is either.

>     * Firefox 47 Release Notes (to be kept for historical reasons?)

Yes, plus this mentions Silverlight in the context of migrating of it, so I think that is fine anyway.

>     * Cross-browser audio basics on MDN

I've deleted the "Fallbacks" section that talks about Flash and Silverlight.

>     * Audio and Video Delivery on MDN

Updated the wording on here to talk about Flash and Silverlight usage being historic, and make it clear that these p[lugins are obsolete.

>     * HTTP headers on MDN

It is useful to still describe this header so that readers can find out what it is if they meet it in the wild, but I've added some notes in parens to make it clear that flash/silverlight are obsolete.

>     * Accessible multimedia on MDN

Removed mentions of Flash.

>     * Video and audio content on MDN

Still useful in the historical context, but I've added a note to say that both are now obsolete.

>     * Apps CSP on MDN

I'm not going to touch this. It's totally obsolete and in the archive.
